ui.py

```diff
@@ -38,17 +38,16 @@ def __init__(self, quiz_brain: QuizBrain):
                                    bd=0,
                                    padx=20,
                                    pady=20,
-                                   command=self.check_if_false)
+                                   command=lambda x="False": self.display_result(x))
```

Comment:
tu powinine być odpalany validator, dopiero to co zwroci powinno miec wplyw na to co zostanie pokazane

Reply:
Nie umiem pozyskiwać wartości zmiennej z funkcji odpalanej przez parametr command= - jak to zrobić?

Reply:
To zależy ;) Jesłi mowimy o parametrze command Buttona, to on jest bezparametryczny AFAIK. Czyli TU musisz odpalać funkcję lambdą z podaniem True, albo False. Sek tylko w tym, jaką funkcję. To powinna być funkcją walidująca z brain, a nie z UI.

Reply:
Próba mikrofonu ;)